_field-types/supported-field-types/star-tree.md

```diff
@@ -105,17 +105,20 @@ PUT logs
 }
 ```
 
+## STIX configuration options
 
+You can customize your STIX implementation using the following `config` options.
 
-## Star-tree mapping parameters
+| Parameter  | Description   | 
+| :--- | :--- |
+| `mappings` | A list of [mapping](#mapping-parameters) parameters. Required.
```

Comment:
This is inaccurate, the config is part of mapping. And config contains 4 parameters , 2 simple ones max leaf docs and skip star node. And 2 complex ones, ordered dimensions and metrics.

If needed we can give a brief about ordered dimensions and metrics in the table here. And then we can continue with separate sections for ordered dimensions and metrics as already present